packages/myst-directives/src/quote.ts

```diff
@@ -32,6 +32,7 @@ export const blockQuoteDirective: DirectiveSpec = {
     const container = {
       type: 'container',
       kind: 'quote',
+      label: data.options?.label as string | undefined,
```

Comment:
Can you follow the pattern here:
https://github.com/executablebooks/mystjs/blob/main/packages/myst-directives/src/figure.ts#L105

const { label, identifier } = normalizeLabel(data.options?.label as string | undefined) || {};

Reply:
Oh whoops yes, I was deep in `rg` when I added that particular string!